plugins/modules/system/homectl.py

```diff
@@ -0,0 +1,618 @@
+        self.name = module.params['name']
+        self.password = module.params['password']
+        self.storage = module.params['storage']
+        self.disksize = module.params['disksize']
+        self.resize = module.params['resize']
+        self.realname = module.params['realname']
+        self.realm = module.params['realm']
+        self.email = module.params['email']
+        self.location = module.params['location']
+        self.iconname = module.params['iconname']
+        self.homedir = module.params['homedir']
+        self.imagepath = module.params['imagepath']
+        self.uid = module.params['uid']
+        self.gid = module.params['gid']
+        self.umask = module.params['umask']
+        self.memberof = module.params['memberof']
+        self.skeleton = module.params['skeleton']
+        self.shell = module.params['shell']
+        self.environment = module.params['environment']
+        self.timezone = module.params['timezone']
+        self.locked = module.params['locked']
+        self.passwordhint = module.params['passwordhint']
+        self.sshkeys = module.params['sshkeys']
+        self.language = module.params['language']
+        self.notbefore = module.params['notbefore']
+        self.notafter = module.params['notafter']
+        self.mountopts = module.params['mountopts']
+
+        self.result = {}
+
+    # Cannot run homectl commands if service is not active
+    def homed_service_active(self):
+        is_active = True
+        cmd = ["systemctl", "show", "systemd-homed.service", "-p", "ActiveState"]
+        rc, show_service_stdout, stderr = self.module.run_command(cmd)
+        if rc == 0:
+            state = show_service_stdout.rsplit("=")[1]
+            if state.strip() != "active":
+                is_active = False
+        return is_active
+
+    def user_exists(self):
+        # Get user properties if they exist in json
+        exists = False
+        cmd = [self.module.get_bin_path('homectl', True)]
+        cmd.append("inspect")
+        cmd.append(self.name)
+        cmd.append("-j")
+        rc, stdout, stderr = self.module.run_command(cmd, use_unsafe_shell=True)
+        if rc == 0:
+            exists = True
+        return exists
+
+    def create_user(self):
+        record = self.create_json_record(create=True)
+        cmd = [self.module.get_bin_path('homectl', True)]
+        cmd.append("create")
+        cmd.append("--identity=-")  # Read the user record from standard input.
+        return(self.module.run_command(cmd, data=record, use_unsafe_shell=True))
+
+    def _hash_password(self, password):
+        method = crypt.METHOD_SHA512
+        salt = crypt.mksalt(method, rounds=10000)
+        pw_hash = crypt.crypt(password, salt)
+        return pw_hash
+
+    def remove_user(self):
+        cmd = [self.module.get_bin_path('homectl', True)]
+        cmd.append('remove')
+        cmd.append(self.name)
+        return(self.module.run_command(cmd, use_unsafe_shell=True))
```

Comment:
1. There is a space missing after the `return` keyword.
2. The brakets are superfluous.
```suggestion
        return self.module.run_command(cmd, use_unsafe_shell=True)
```
Same for similar calls.